docs/usage.rst

```diff
@@ -4,9 +4,135 @@ Usage
 Creating Virtual Machines
 -------------------------
 
-Create vms
+Follow the steps carefully so that you dont get any error while creating virtual machine. The steps are very easy only need is to follow them in order.
+
+++++++
+Step 1
+++++++
+
+First you need to create compute resource.
+
+To do so click on Infrastructure -> Compute Resource
+
+* Fill all the details
+* Make sure there is no repeatation of name, remote system is reachable from your system and root password is correct.
+* All the fields have validation.
+
+++++++
+Step 2
+++++++
+
+Once done with compute resource move to profiles.
+
+Click on Infrastructure -> Profile
+
+* Name your Profile, fill the fields of RAM, VCPUS and Disk Space
+* Name should not get repeated.
+
+Once you click *Submit* Profile gets added to database and is visible in **View Existing** Section.
+
+++++++
+Step 3
```

Comment:
mention here that adding packages is optional. If the user wants to add then only follow step 3 else skip them